### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by cloud9350** [AndreRH/hangover#7]
```
gureumi@ubuntu:~/hangover$ make -f Makefile.android
make -C build.android/wine-host
make[1]: Entering directory '/home/gureumi/hangover/build.android/wine-host'
make[2]: Entering directory '/home/gureumi/hangover/build.android/wine-host/libs/wpp'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/gureumi/hangover/build.android/wine-host/libs/wpp'
make[2]: Entering directory '/home/gureumi/hangover/build.android/wine-host/libs/port'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/gureumi/hangover/build.android/wine-host/libs/port'
make[2]: Entering directory '/home/gureumi/hangover/build.android/wine-host/libs/wine'
version=`(GIT_DIR=../../../../wine/.git git describe HEAD 2>/dev/null || echo "wine-3.12") | sed -n -e '$s/\(.*\)/const char wine_build[] = "\1";/p'` && (echo $version | cmp -s - version.c) || echo $version >version.c || (rm -f version.c && exit 1)
make[2]: Leaving directory '/home/gureumi/hangover/build.android/wine-host/libs/wine'
make[2]: Entering directory '/home/gureumi/hangover/build.android/wine-host/include'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/gureumi/hangover/build.android/wine-host/include'

~~~ skip line ~~~

make[2]: Entering directory '/home/gureumi/hangover/build.android/wine-host/dlls/windowscodecsext'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/gureumi/hangover/build.android/wine-host/dlls/windowscodecsext'
make[2]: Entering directory '/home/gureumi/hangover/build.android/wine-host/dlls/wineandroid.drv'
(test -d assets && \
  rm -f assets/files.sum assets/sums.sum && \
  sha256sum `find assets -type f -print` | sed 's/ assets\// /' >files.sum && \
  sha256sum files.sum >sums.sum && \
  mv files.sum sums.sum assets) || rm -rf assets
gradle -q assembleDebug && mv build/outputs/apk/wine-debug.apk .

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':createIcon-hdpi'.
> A problem occurred starting process 'command 'rsvg-convert''

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
Makefile:189: recipe for target 'wine-debug.apk' failed
make[2]: *** [wine-debug.apk] Error 1
make[2]: Leaving directory '/home/gureumi/hangover/build.android/wine-host/dlls/wineandroid.drv'
Makefile:8082: recipe for target 'dlls/wineandroid.drv' failed
make[1]: *** [dlls/wineandroid.drv] Error 2
make[1]: Leaving directory '/home/gureumi/hangover/build.android/wine-host'
Makefile.android:132: recipe for target 'build.android/wine-host/.built' failed
make: *** [build.android/wine-host/.built] Error 2
gureumi@ubuntu:~/hangover$
```

**Comment by AndreRH** [comment]
Do you have rsvg-convert installed?

**Comment by cloud9350** [comment]
O.o It's need librsvg2-bin
```apt-get install librsvg2-bin```

**Comment by cloud9350** [comment]
```
gureumi@DESKTOP-U63LTE8:~$ cd hangover
gureumi@DESKTOP-U63LTE8:~/hangover$ make -f Makefile.android
make -C build.android/qemu
make[1]: Entering directory `/home/gureumi/hangover/build.android/qemu'
  CXX     disas/arm-a64.o
In file included from /home/gureumi/ndk/include/c++/4.9.x/type_traits:390:0,
                 from /home/gureumi/ndk/include/c++/4.9.x/memory:643,
                 from /home/gureumi/ndk/include/c++/4.9.x/list:173,
                 from /home/gureumi/hangover/qemu/disas/libvixl/vixl/a64/decoder-a64.h:30,
                 from /home/gureumi/hangover/qemu/disas/libvixl/vixl/a64/disasm-a64.h:33,
                 from /home/gureumi/hangover/qemu/disas/arm-a64.cc:25:
/home/gureumi/ndk/include/c++/4.9.x/cstddef:44:25: fatal error: stddef.h: No such file or directory
 #include_next <stddef.h>
                         ^
compilation terminated.
winegcc: aarch64-linux-android-g++ failed
make[1]: *** [disas/arm-a64.o] Error 2
make[1]: Leaving directory `/home/gureumi/hangover/build.android/qemu'
make: *** [build.android/qemu/x86_64-windows-user/qemu-x86_64.exe.so] Error 2
gureumi@DESKTOP-U63LTE8:~/hangover$
```
🤔

**Comment by AndreRH** [comment]
Should be fixed by 562a998cadfe43473cd48185a83ddc28cb96d0a0
